tests/data/markdown/basic.toml

```diff
@@ -76,8 +76,8 @@ ___
sk emphasized</em>, <strong>underscore bold</strong> and <em>underscore italic</em> strings.</p>\n<p>This is a <s>final</s>sample paragraph that has an <em><strong>asterisk bold italic</strong></em> string and an <em><strong>underscore bold italic</strong></em> string.<br />\nIt also has a newline break here!!!</p>\n<p>Here are examples of bold and emphasized text depending on the placement of underscores/asterisks:</p>\n<p>__Bold without closure does not work<br />\n**Bold without closure does not work<br />\n_Emphasis without closure does not work<br />\n*Emphasis without closure does not work</p>\n<p><strong>Bold without closure<br />\non the same line<br />\ncarries forward to<br />\n<s>this is text that should strike off</s><br />\nmultiple consecutive lines</strong></p>\n<p><strong>Bold without closure<br />\non the same line<br />\ncarries forward to<br />\nmultiple consecutive lines</strong></p>\n<p><em>Emphasis without closure<br />\non the same line<br />\ncarries forward to<br />\nmultiple consecutive lines</em></p>\n<p><em>Emphasis without closure<br />\non the same line<br />\ncarries forward to<br />\nmultiple consecutive lines</em></p>\n<h3 id=\"h:horizontal-rules\">Horizontal Rules <a class=\"header-anchor\" href=\"#h:horizontal-rules\">¶</a></h3>\n<hr />\n<p>Above is a horizontal rule using hyphens.<br />\n<s>This is text that should strike off</s></p>\n<hr />\n<p>Above is a horizontal rule using asterisks.<br />\nBelow is a horizontal rule using underscores.</p>\n<hr />\n<h2 id=\"h:links\">Links <a class=\"header-anchor\" href=\"#h:links\">¶</a></h2>\n<p>Here is a link to <a href=\"https://hasgeek.com\">hasgeek</a></p>\n<p><a href=\"https://github.com/hasgeek/funnel/\" title=\"Hasgeek\">Link to funnel with the title 'Hasgeek'</a></p>\n<p>Autoconverted link <a href=\"https://github.com/hasgeek\">https://github.com/hasgeek</a> (will autoconvert if linkify is on)</p>\n<hr />\n<h2 id=\"h:markdown-it-typography\">Markdown-it typography <a class=\"header-anchor\" href=\"#h:markdown-it-typography\">¶</a></h2>\n<p>The results of the below depend on the typographer options enabled by us in the <code>markdown-it-py</code> parser, if <code>typographer=True</code> has been passed to it.</p>\n<p><strong>The below should convert if <code>replacements</code> has been enabled.</strong></p>\n<p>(c) (C) (r) (R) (tm) (TM) (p) (P) +-<br />\ntest.. test... test..... test?..... test!....<br />\n!!!!!! ???? ,,  -- ---</p>\n<p><strong>The below should convert the quotes if <code>smartquotes</code> has been enabled.</strong></p>\n<p>&quot;Smartypants, double quotes&quot; and 'single quotes'</p>\n<hr />\n<h2 id=\"h:blockquotes\">Blockquotes <a class=\"header-anchor\" href=\"#h:blockquotes\">¶</a></h2>\n<blockquote>\n<p>Blockquotes can also be nested...</p>\n<blockquote>\n<p>...by using additional greater-than signs right next to each other...</p>\n<blockquote>\n<p>...or with spaces between arrows.</p>\n</blockquote>\n</blockquote>\n</blockquote>\n<hr />"
```

Comment:
These should use multiline strings for readability.

Reply:
Can I post an issue for this and resolve it separately?

Reply:
As discussed, this will be addressed in #1506